.github/workflows/claude-ci-autofix.yml

```
@@ -32,56 +30,62 @@ jobs:
-            const run_id = ${{ github.event.workflow_run.id }};
+            const triggering_run_id = ${{ github.event.workflow_run.id }};
+            core.setOutput('triggering_run_id', triggering_run_id);
+            console.log(`Triggering workflow run ID: ${triggering_run_id}`);
+
             const jobs = await github.rest.actions.listJobsForWorkflowRun({
               owner: context.repo.owner,
               repo: context.repo.repo,
-              run_id: run_id,
+              run_id: triggering_run_id,
             });
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)